### PR TITLE
fix(db-migration): fix ss58 prefix migration for first-priority data

### PR DESCRIPTION
**Pull request by AndreiNavoichyk**
Migrated from callback-based approach to async/await approach.

### CODE REVIEW

File: .github/workflows/dev.yaml [hunk 1]
```diff
@@ -29,19 +29,3 @@ jobs:
       repository: stats-crawler
       file: ./backend/docker/crawler/Dockerfile
     secrets: inherit
-
```
Comment (AndreiNavoichyk) [inline .github/workflows/dev.yaml]: Revert

File: db/migrations/20230227122704-3-ss58-prefix.js [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+exports.setup = function (options, seedLink) {
+    dbm = options.dbmigrate;
+    type = dbm.dataType;
+    seed = seedLink;
+    Promise = options.Promise;
+};
+
+// Migration script for Account Table
+const convertAccountTable = async (db, ss58Format) => {
+    const deleteResult = await executeDbRunSqlAsPromise(db, "DELETE FROM account WHERE account_id LIKE '6%'");
```
Comment (AndreiNavoichyk) [inline db/migrations/20230227122704-3-ss58-prefix.js]: Remove

File: db/migrations/20230227122704-3-ss58-prefix.js [hunk 3]
```diff
@@ -0,0 +1,249 @@
...
+    }
+
+    log = () => {
+        this.currentRow++;
+        const {currentRow, totalRows, tableName} = this;
+
+        if (currentRow === totalRows) {
+            console.log(`✅ Done for ${tableName} table`);
+        } else {
+            if (currentRow % 150 === 0) {
```
Comment (AndreiNavoichyk) [inline db/migrations/20230227122704-3-ss58-prefix.js]: Make less logs

File: db/migrations/20230227122704-3-ss58-prefix.js [hunk 4]
```diff
@@ -0,0 +1,253 @@
...
+        'era_points',
+        'era_relative_performance',
+        'era_self_stake',
+        'era_vrc_score',
+    ];
+
+    for (let i = 0; i < tables.length; i++) {
+        const table = tables[i];
+
+        const deleteResult = await executeDbRunSqlAsPromise(db, `DELETE FROM ${table} WHERE stash_address LIKE '6%'`);
```
Comment (AndreiNavoichyk) [inline db/migrations/20230227122704-3-ss58-prefix.js]: Remove

File: db/migrations/20230227122704-3-ss58-prefix.js [hunk 5]
```diff
@@ -0,0 +1,253 @@
...
+        return keyring.encodeAddress(
+            keyring.decodeAddress(address),
+            ss58);
+    }
+};
+
+const executeDbRunSqlAsPromise = (db, sqlQuery) => {
+    return new Promise(function (resolve, reject) {
+        db.runSql(sqlQuery, (error, result) => {
+            if (error) {
```
Comment (AndreiNavoichyk) [inline db/migrations/20230227122704-3-ss58-prefix.js:231]: Add to log `sqlQuery`

File: db/migrations/20230227122704-3-ss58-prefix.js [hunk 6]
```diff
@@ -0,0 +1,253 @@
...
+
+            await executeDbRunSqlAsPromise(db, queryString);
+
+            logger.log();
+        }
+    }
+};
+
+const convertRankingTable = async (db, ss58Format) => {
+    const deleteResult = await executeDbRunSqlAsPromise(db, "DELETE FROM ranking WHERE stash_address LIKE '6%' OR controller_address LIKE '6%'");
```
Comment (AndreiNavoichyk) [inline db/migrations/20230227122704-3-ss58-prefix.js]: Delete

File: db/migrations/20230227122704-3-ss58-prefix.js (hunk 5 above)
Comment (shamilkhan) [reply db/migrations/20230227122704-3-ss58-prefix.js:231]: Done. Added 
`console.log(error);`

File: db/migrations/20230227122704-3-ss58-prefix.js (hunk 4 above)
Comment (shamilkhan) [reply db/migrations/20230227122704-3-ss58-prefix.js]: Done

File: db/migrations/20230227122704-3-ss58-prefix.js (hunk 6 above)
Comment (shamilkhan) [reply db/migrations/20230227122704-3-ss58-prefix.js]: Done

File: db/migrations/20230227122704-3-ss58-prefix.js (hunk 3 above)
Comment (shamilkhan) [reply db/migrations/20230227122704-3-ss58-prefix.js]: Done. 
Screenshot 2023-03-22 at 14 13 22

File: db/migrations/20230227122704-3-ss58-prefix.js (hunk 2 above)
Comment (shamilkhan) [reply db/migrations/20230227122704-3-ss58-prefix.js]: Done

File: .github/workflows/dev.yaml (hunk 1 above)
Comment (shamilkhan) [reply .github/workflows/dev.yaml]: Done.

File: db/migrations/20230227122704-3-ss58-prefix.js (hunk 5 above)
Comment (AndreiNavoichyk) [reply db/migrations/20230227122704-3-ss58-prefix.js:231]: Please, add also `sqlQuery`.

File: db/migrations/20230227122704-3-ss58-prefix.js (hunk 3 above)
Comment (AndreiNavoichyk) [reply db/migrations/20230227122704-3-ss58-prefix.js]: Let's do step equal to `10`.

File: db/migrations/20230227122704-3-ss58-prefix.js (hunk 5 above)
Comment (shamilkhan) [reply db/migrations/20230227122704-3-ss58-prefix.js:231]: Done.

**Review by AndreiNavoichyk — COMMENTED** [review]
Approved

**Pull request merged**